controllers/cluster/controllers/cluster_controller.go

```diff
@@ -18,9 +18,16 @@ package controllers
 
 import (
 	"context"
+	"errors"
 	"fmt"
+	"path/filepath"
 	"time"
 
+	"github.com/labring/sealos/pkg/utils/logger"
```

Comment:
Controllers can direct use kubebuilder‘s reconcile logger to log informations, there is no such need to use sealos's `pkg/logger`, witch will cause package all `sealos/pkg` code, seems too large for controller.